packages/test/local-server-stress-tests/src/ddsModels.ts

```diff
@@ -0,0 +1,214 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	done,
+	type AsyncGenerator,
+	type AsyncReducer,
+} from "@fluid-private/stochastic-test-utils";
+import {
+	DDSFuzzModel,
+	DDSFuzzTestState,
+	Client as DDSClient,
+} from "@fluid-private/test-dds-utils";
+import { fluidHandleSymbol } from "@fluidframework/core-interfaces";
+import { assert, isObject } from "@fluidframework/core-utils/internal";
+import type {
+	IChannel,
+	IChannelFactory,
+} from "@fluidframework/datastore-definitions/internal";
+// eslint-disable-next-line import/no-internal-modules
+import { baseMapModel, baseDirModel } from "@fluidframework/map/internal/test";
+import { toFluidHandleInternal } from "@fluidframework/runtime-utils/internal";
+import {
+	baseSharedStringModel,
+	baseIntervalModel,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/sequence/internal/test";
+
+import {
+	LocalServerStressState,
+	makeUnreachableCodePathProxy,
+	Client,
+} from "./localServerStressHarness";
+
+export function repeatFactoryAsync<T, TState = void>(
+	factory: () => AsyncGenerator<T, TState>,
```

Comment:
TIL we have our own non-standard version of AsyncGenerator that isn't even iterable :( Is there a world where we can get away from this?